partcad/src/partcad/ai_feature_file.py

```
@@ -0,0 +1,92 @@
+                raise Exception(f"Failed to download the file {self.url}")
+
+            # Strip the query string from the URL to get the extension
+            if "?" in lower_url:
+                lower_url = lower_url[: lower_url.index("?")]
+        else:
+            self.filename = self.url
+
+        if lower_url.endswith(".png"):
+            self.is_image = True
+            self.suffix = ".png"
+        elif lower_url.endswith(".jpg") or lower_url.endswith(".jpeg"):
```

Ruff (0.8.2)

Call `endswith` once with a `tuple`

Merge into a single `endswith` call

(PIE810)